### PR TITLE
feat: pipeline for production and development

### PR DESCRIPTION
**Pull request by amordzon**
## Description
created pipeline for production and development

## Clickup Ticket

https://app.clickup.com/t/869421uxa

### CODE REVIEW

**Review by jtowarnicka — APPROVED** [review]
lgtm

File: .github/workflows/DEVELOP-CI-CD.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: DEVELOP-CI-CD
+
+on:
+  push:
+    branches:
+      - develop
+
+jobs:
+  release:
+    needs: [run-unit-tests]
```
Comment (jzapiorkowski) [inline .github/workflows/DEVELOP-CI-CD.yml]: remove `needs` property heve

File: .github/workflows/DEVELOP-CI-CD.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
+name: DEVELOP-CI-CD
```
Comment (jzapiorkowski) [inline .github/workflows/DEVELOP-CI-CD.yml]:
```suggestion
name: DEVELOP-CD
```

File: .github/workflows/PROD-CI-CD.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
+name: PROD-CI-CD
```
Comment (jzapiorkowski) [inline .github/workflows/PROD-CI-CD.yml]:
```suggestion
name: PROD-CD
```

File: .github/workflows/PROD-CI-CD.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+on:
+  push:
+    branches:
+      - production
+
+jobs:
+  release:
+    needs: [run-unit-tests]
```
Comment (jzapiorkowski) [inline .github/workflows/PROD-CI-CD.yml]: remove `needs` property here

**Pull request merged**